src/adapters/convex/products/staking/convexStakingAdapter.ts

```diff
@@ -55,6 +61,16 @@ export class ConvexStakingAdapter
     }
   }
 
+  @AddClaimableRewards({ rewardAdapterIds: ['extra-reward', 'rewards'] })
+  getPositions(input: GetPositionsInput): Promise<ProtocolPosition[]> {
+    return super.getPositions(input)
+  }
+
+  @AddClaimedRewards({ rewardAdapterIds: ['extra-reward', 'rewards'] })
+  getWithdrawals(input: GetEventsInput): Promise<MovementsByBlock[]> {
+    return super.getWithdrawals(input)
+  }
```

Comment:
Add claim**ed** rewards to withdrawals

This ensures they are included in our pnl calculations